src/renderer/components/playlist-info/playlist-info.js

```diff
@@ -412,6 +421,41 @@ export default defineComponent({
       showToast(this.playlistDeletionDisabledLabel)
     },
 
+    handlePlaylistExport: async function () {
+      const dateStr = getTodayDateStrLocalTimezone()
+      const title = this.selectedUserPlaylist.playlistName.replaceAll(' ', '_').replaceAll(/["%*/:<>?\\|]/g, '_')
```

Comment:
Non-blocking: Maybe we should merge the logic here with the logic for replacing the names of files created for screenshots? https://github.com/FreeTubeApp/FreeTube/blob/94fe62aff4c5b721033090a896738bd8006d4c3d/src/renderer/helpers/utils.js#L565